src/django/api/management/commands/delete_contributor.py

```diff
@@ -0,0 +1,42 @@
+import sys
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from allauth.account.models import EmailAddress
+
+from api.models import User, Contributor
+
+
+class Command(BaseCommand):
+    help = 'Given a User email address, delete the User and also delete the ' \
+           'associated Contributor record.'
+
+    def add_arguments(self, parser):
+        group = parser.add_argument_group('required arguments')
+        group.add_argument('-e', '--email',
+                           required=True,
+                           help='The user\'s email address')
+
+    def handle(self, *args, **options):
+        email = options['email']
+
+        try:
+            user = User.objects.get(email=email)
+            contributor = Contributor.objects.get(admin=user)
```

Comment:
We should consider moving the some exception handling up to this contributor fetch. It is possible to have a user account without a related contributor. If that is the case, we still want to delete the user account.

Reply:
Good point. I updated this in 
26f33df by adding separate nested try blocks for the Contributor and EmailAddress objects.